start_esp/start_esp.py

```diff
@@ -791,6 +795,20 @@ def make_argparser():
         default=None,
         help=argparse.SUPPRESS)
 
+    parser.add_argument('--log_request_headers', default=None, help='''
+        Log corresponding request headers into Google cloud console through
+        service control, separated by comma. Example, when
+        --log_request_headers=foo,bar, endpoint log will have
+        request_headers: foo=foo_value;bar=bar_value if values are available;
```

Comment:
Another option is to have separate field for each header, such as

request_header_foo=foo_value
request_header_bar=bar_value

Maybe this is easier to create a query for searching the log.   But I am not sure since I never done a log query before
Maybe request_headers: foo=foo_value;bar=bar_value can be easily queried too.

Hi @inklesspen  what do you think?

Reply:
I don't know which is easier to query for either...
Who asked for this feature originally? I know you opened the issue.

Reply:
It is hard to have separated fields for each header, since it needs to be added into ReportRequestInfo struct under info.h

Reply:
OK,  it is fine to have comma separated multiple headers in one field then.